packages/nextjs/codegen.ts

```diff
@@ -0,0 +1,21 @@
+import { CodegenConfig } from "@graphql-codegen/cli";
+
+export const localHasuraEndpoint = "http://localhost:8080/v1/graphql";
+
+const config: CodegenConfig = {
+  schema: localHasuraEndpoint,
```

Comment:
This should maybe try read from the env var first, it might be missed by a user not knowing its set here too

Reply:
Unfortunately it wasn't working with the graphql codegen command! I tried 👍🏼

Reply:
Ok I've added dotenv and reading from the next js .env.local 👍🏼